src/components/ElixirFooter/__snapshots__/test.js.snap

```diff
@@ -2,76 +2,100 @@
 
 exports[`<ElixirFooter /> should render 1`] = `
 <div
-  className="row columns"
+  className="vf-u-fullbleed elixir-footer"
```

Comment:
Forgot to mention... I'm so glad of this change, as this fixes the case where the elixir header ends up over the filters, not only was ugly but also a bit of a pain on the puppeteer script.